pftp/client_handler.go

```diff
@@ -208,6 +237,13 @@ func (c *clientHandler) handleCommand(line string) (r *result) {
 		}
 	}()
 
+	// Hide user password from log
+	if c.command == "PASS" {
```

Comment:
It seems better to divide it into methods.

Reply:
```go
func commandLog(format, line string) {
  command := strings.ToUpper(strings.SplitN(strings.Trim(line, "\r\n"), " ", 2)[0])
  ...
}
```

Reply:
You're right!

I divide the function.

Now user can set commands what needs to secure in config.toml file.

ex) if want to secure logged in user id/password and up/downloaded file name.
```
secure_commands = ["USER", "PASS", "STOR", "RETR"]
```

https://github.com/pyama86/pftp/pull/26/commits/4877eddc5b9eb0233abfbc8d79db440a57d9b92d

Reply:
wow.
great!but,secure_commands seems to be a good constant！

Reply:
I fixed it.

Now secure commands is const and only masking "PASS" command!
https://github.com/pyama86/pftp/pull/26/commits/342b94422484f4f6ef2317c16c7d3c0a563b6604